apps/analytics/views.py

```diff
@@ -211,3 +207,58 @@ def get_last_submission_datetime_analysis(request, challenge_pk, challenge_phase
     except:
         response_data = {'error': 'Bad request. Please try again later!'}
         return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+@throttle_classes([UserRateThrottle])
+@api_view(['GET', ])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_challenge_phase_submission_analysis(request, challenge_pk, challenge_phase_pk):
+    """
+    API to fetch
+    1. The total submissions count in a challenge phase.
+    2. The participant team count in a challenge phase.
+    3. Submission count for Submitted/Running/Failed/Cancelled/Finished/Submitting submissions.
```

Comment:
Replace with `3. Number of submissions with status a)Submitting, b)Submitted, c)Running, d)Failed, e)Cancelled, f)Finished status`